test-crawler/collector/collector.go

```diff
@@ -88,3 +93,35 @@ func listTestFiles(root string, ignore []string) (files []string, err error) {
 
 	return nil, nil
 }
+
+// listGoFilesInFolder returns list of all .go files, excluding _test.go
+// Root folder will represent a system for which the methods will be crawled.
+func ListGoFilesInFolder(root string, ignore []string) (system string, files []string, err error) {
```

Comment:
I assume `ignore` is meant to be used in the future?

Reply:
Yes, in the future that function will be expanded for different purposes, accepting what file types should be included, excluded, ignored, etc...

Reply:
Please add that to the comment for clarification.